package.json

```diff
@@ -5,7 +5,7 @@
     "build": "turbo build --concurrency 16",
     "build:examples": "turbo build --filter=@example/*",
     "changeset": "changeset",
-    "clean": "turbo clean",
+    "clean": "rm -rf packages/*/dist packages/*/*.tsbuildinfo; turbo clean",
```

Comment:
pls revert on this pr - this can have side effects. if you think it's useful let's do it on a separate pr but i've played with similar things at some point and they can lead to issues w/ turbo locally

Reply:
oops yep this slipped into wrong PR